src/content/pages/index.md

```diff
@@ -84,38 +69,6 @@ tokenHolderRole:
       variant: INTERNAL_OUTLINE
   rowReverse: true
   bgColor: "#3c3c3c"
-migrationInfo:
-  title: Do you own KEEP or NU?
-  description: Keep and NuCypher merged to form the Threshold Network. Upgrade
-    your tokens to T!
-  image: /images/legacy-tokens.png
-  buttons:
-    - label: Upgrade tokens
-      url: https://dashboard.threshold.network/upgrade
-      variant: EXTERNAL_OUTLINE
-  bgColor: "#141414"
-  rowReverse: false
-tacoRole:
-  title: Build uncensorable e2ee into your dapp
-  description: TACo is the only end-to-end encryption plugin that is end-to-end decentralized
-  image: /images/taco-section-image.svg
-  buttons:
-    - label: Learn More
-      url: /build/taco
-      variant: INTERNAL_SOLID
-    - label: Quickstart
-      url: https://docs.threshold.network/app-development/threshold-access-control-tac/quickstart-testnet
-      variant: EXTERNAL_OUTLINE
```

Comment:
During the transition period, let's leave a reduced TACo section that links to taco.build. This can be moved to the foot of the homepage if necessary. This section should avoid references to disruptive plans and can simply retain the following:

title: **Build uncensorable e2ee into your dApp**
description: **TACo** is the only end-to-end encryption plugin that is end-to-end decentralized
  buttons:
    - label: taco.build
      url: https://taco.build
      variant: INTERNAL_SOLID
    - label: Docs
      url: https://docs.taco.build